### PR TITLE
Add Support for Other Bucket / Other Bucket Key Parameters on Filters Aggregation

### PR DESCRIPTION
**Pull request by mtdavidson**
Add ability to set `other_bucket` and `other_bucket_key` parameters on filters aggregation. 

https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-filters-aggregation.html#_literal_other_literal_bucket

### CODE REVIEW

File: lib/Elastica/Aggregation/Filters.php [hunk 1]
```diff
@@ -59,6 +59,30 @@ public function addFilter(AbstractQuery $filter, $name = null)
     }
 
     /**
+     * @param bool $other_bucket
+     *
+     * @return $this
+     */
+    public function setOtherBucket($other_bucket)
+    {
+        if (!is_bool($other_bucket)) {
```
Comment (ruflin) [inline lib/Elastica/Aggregation/Filters.php]: We use lowerUpper case for the variable names across Elastica. Could you change it to $otherBucket? Same for the other variable names in this PR

Comment (mtdavidson) [inline lib/Elastica/Aggregation/Filters.php]: Thanks for the feedback I've updated these variables.

**Comment by mtdavidson** [comment]
Thanks for the feedback @ruflin I've done the variable fix, updated the change log and added the functional tests.

**Comment by ruflin** [comment]
@mtdavidson Thanks, merged.

**Pull request merged**